tap_eloqua/__init__.py

```diff
@@ -51,6 +51,8 @@ def parse_args(required_config_keys):
     -d,--discover   Run in discover mode
     -p,--properties Properties file: DEPRECATED, please use --catalog instead
     --catalog       Catalog file
+    -dev, --dev     Runs the tap in dev mode
```

Comment:
Generally, we use `-` for single character switch options whereas `--` for descritive switch options. Should we follow that? In that we will have single switch `--dev` for dev mode since `-d` is already used for discovery.

Reply:
replaced this with singer-python implementation
these changes are now dependant on P.R https://github.com/singer-io/singer-python/pull/158